src/seedu/addressbook/util/FilePathUtil.java

```diff
@@ -0,0 +1,66 @@
+package seedu.addressbook.util;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * This is a utility class for file path validation.
+ */
+public class FilePathUtil {
+    private static final int EXTENSION_INVALID_INDEX = 0;
+
+    /**
+     * Returns true if the given file is acceptable.
+     * The file path is acceptable if it does not contain any reserved characters <>:"/|?*,
```

Comment:
We are checking if the directory exists, the file name is acceptable to the operating system, and has an extension, right?